src/target/riscv/riscv.c

```diff
@@ -5360,10 +5360,11 @@ riscv_insn_t riscv_read_debug_buffer(struct target *target, int index)
 	return r->read_debug_buffer(target, index);
 }
 
-int riscv_execute_debug_buffer(struct target *target)
+int riscv_execute_debug_buffer(struct target *target,
+	unsigned int *abstractcs_err)
```

Comment:
Same as above - consider calling the last parameter `cmderr` so that the names are unified.

```suggestion
int riscv_execute_debug_buffer(struct target *target,
	unsigned int *cmderr)
```

Reply:
addressed